src/System.Linq.Expressions/src/System/Linq/Expressions/DebugViewWriter.cs

```diff
@@ -1014,7 +1014,7 @@ protected internal override Expression VisitBlock(BlockExpression node)
             // last expression's type in the block.
             if (node.Type != node.GetExpression(node.ExpressionCount - 1).Type)
             {
-                Out(String.Format(CultureInfo.CurrentCulture, "<{0}>", node.Type.ToString()));
+                Out(string.Format(CultureInfo.CurrentCulture, "<{0}>", node.Type.ToString()));
```

Comment:
Separate issue but noticed a lot of usage of `CurrentCulture` in this file. We may want to revisit this.